magstock/__init__.py

```diff
@@ -0,0 +1,75 @@
+from uber.common import *
+from magstock._version import __version__
+
```

Comment:
This is where we perform our template and file overriding:
* Anything in the ``magstock/static`` directory will be loaded in place of anything in the ``uber/static`` directory.  The main thing we want to override is stuff in the ``static/theme`` subdirectory, but theoretically we can override anything.
* Anything in the ``magstock/templates`` directory will be loaded as a template in place of the same template from the ``magstock/templates`` directory.

The main template that we expect every plugin to override is ``regextras.html``.  This is an empty template which exists only to be overridden.  We have a single ``regform.html`` template which is included in all of the following places:
* the main preregistration form
* the prereg confirmation page
* the badge transfer form
* the form for claiming an unassigned group registration
* the reg admin form
* the at-the-door registration form

By overriding ``regextras.html`` we can add extra form fields.  We can also remove existing ones with Javascript as well, e.g. if there's some form field you don't care about.  (For example, maybe an event doesn't care about asking "How did you find our event?")